cmds/netcat/netcat.go

```diff
@@ -1,37 +1,178 @@
-// Copyright 2012 the u-root Authors. All rights reserved
-// Use of this source code is governed by a BSD-style
-// license that can be found in the LICENSE file.
-
-/*
-netcat connects to a place and sends data to it and from it.
-*/
-
 package main
```

Comment:
we need a license statement. You can't just remove one and not have something there.